lineup.py

```diff
@@ -1,8 +1,15 @@
 from bs4 import BeautifulSoup
+import easyocr
 
 
 def find_artists(file, find_all: str, class_: str):
     with open(file) as fp:
         soup = BeautifulSoup(fp, "html.parser")
 
     return soup.find_all(find_all, class_=class_)
+
+def find_easyocr_lineup(image_src):
```

Comment:
@chunswu as an extension later we could use type hints to say that `image_src` should be a string type, and we could also add a docstring, but for now this is fine.

Reply:
@DaveParr will add in more documentation later on in the project.